packages/rum-core/src/common/constants.js

```diff
@@ -110,6 +110,7 @@ const TRANSACTION_END = 'transaction:end'
 const CONFIG_CHANGE = 'config:change'
 const QUEUE_FLUSH = 'queue:flush'
 const QUEUE_ADD_TRANSACTION = 'queue:add_transaction'
+const TRANSACTION_DISCARD = 'transaction:discard'
```

Comment:
should we rename it as trasaction:ignore/skip ? mainly suggesting because ignored also aligns with our config settings.

Reply:
yes, it makes sense 🚀